CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # ChangeLog
 
+## **next**
+
+### Bug fixes
+
+- [Host/Dashboard] Fix accent of tag list (PR [#8731](https://github.com/vatesfr/xen-orchestra/pull/8731))
```

Comment:
- Keep it user-oriented: `accent` => `color`
- We don't edit `CHANGELOG.md` directly, `CHANGELOG.unreleased.md` will be merged into it during the release. However, since it's also visible in XO Lite, you can add it to XO Lite's changelog